example/src/redux/store.js

```diff
@@ -0,0 +1,26 @@
+import { createStore, applyMiddleware, compose, combineReducers as CR } from 'redux';
+import { fetchMiddleware, configureMergeState, wrapCombineReducers } from 'redux-recompose';
+import thunk from 'redux-thunk';
+
+import hearthstone from './hearthstone/reducer';
+
+// Breaking in r-r v2.0: need to specify how do we merge (removed coupled seamless-immutable)
+configureMergeState((state, diff) => state.merge(diff));
+
+// Use this function to let invisible reducer override behavior when needed
+const combineReducers = wrapCombineReducers(CR);
```

Comment:
I don't get how this works. Where's the effect the invisible reducer has in the store?

Reply:
I guess it's kind of the point that the effect is "invisible" xD. But I guess I don't understand it yet

Reply:
The main idea here is that `wrapCombineReducers` lets the `invisible` introduce cross-reducer modifications, by overriding how `combineReducers` behaves by default.